src/utils/paOnAuthorizationSuccess.js

```diff
@@ -0,0 +1,64 @@
+import paOnAuthorizationSuccessValidator from '../validators/paOnAuthorizationSuccess';
+
+const credentials = 'include';
+
+const getParameters = (response) => {
+  const body = new URLSearchParams();
```

Comment:
Does this work in IE? I know of, Object.entries have bit us in the past. Looks like URLSearchParams might be limited as well

Reply:
Ah, nice catch. I think made the bad assumption that this would be handled by babel transpilation. It looks like I'll need to update this block to manually construct the parameter string and iterate through the response object.

Reply:
I've updated this method to manually construct query param string and replaced Object.entries with Object.keys.